miro_preprocessor/xml_to_json_converter/run.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+Parse image records from a Miro export and push JSON derivatives to S3.
+
+The output is a file stored in S3 which encodes Miro records as JSON objects,
+one object per line.  This makes it easier for us to run one-off scripts
```

Comment:
I thought we were writing individual JSON files as well?

Reply:
Yes, but not in this PR.